app/src/app/components/contents/explorer/Explorer.tsx

```diff
@@ -1,7 +1,50 @@
-import React from "react";
+import React, {useEffect} from "react";
+import Table from "./Table";
+import {useSelector} from "react-redux";
+import {RootState, useAppDispatch} from "../../../../types";
+import {actions} from "../../../store/packets/thunks";
 
 export default function Explorer() {
+
+    const {packets} = useSelector((state: RootState) => state.packets);
+
+    const dispatch = useAppDispatch();
+
+    useEffect(() => {
+        dispatch(actions.fetchPackets());
+    }, []);
+
+    const headers = [
+        {label: "Name", accessor: "name", sortable: true},
+        {label: "Version", accessor: "version", sortable: true},
+        {label: "Status", accessor: "status", sortable: true},
+        {label: "Parameters", accessor: "parameters", sortable: false}
+    ];
     return (
-        <div>Packet explorer page</div>
+        <div className="content explorer">
+            <div className="small">
+                <span className="d-flex">Packets ({packets.length})</span>
+                <span className="d-flex pb-3">Click on a column heading to sort by field.</span>
+            </div>
+            <div className="content-box">
+                <div className="table-responsive-sm pt-4">
+                    <table data-testid="table" className="table table-hover table-bordered table-sm">
+                        <Table data={packets.slice(0, 5)} headers={headers}/>
```

Comment:
I am assuming this slice index will be dynamically updated in a later PR/feature?

Reply:
Yes, that's just a placeholder in which pagination implementation will resolve.